### ISSUE TITLE
Importing an existing azurerm_monitor_metric_alert resource does not fetch all fields

### ISSUE THREAD

**Issue by xsreality** [terraform-providers/terraform-provider-azurerm#7560]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

Terraform v0.12.25
+ provider.azurerm v2.16.0

### Affected Resource(s)

* `azurerm_monitor_metric_alert`

### Terraform Configuration Files

```hcl
resource "azurerm_monitor_metric_alert" "example" {
  name = "example-alert"
  resource_group_name = azurerm_resource_group.traffic_manager_rg.name
  scopes = [azurerm_traffic_manager_profile.example.id]
  description = "Traffic Manager Endpoint Status by Endpoint"
  severity = 2

  criteria {
    aggregation = "Average"
    metric_name = "ProbeAgentCurrentEndpointStateByProfileResourceId"
    metric_namespace = "Microsoft.Network/trafficManagerProfiles"
    operator = "LessThan"
    threshold = 1
  }

  tags = {}

  action {
    action_group_id = azurerm_monitor_action_group.action_group_team.id
  }
}
```

### Expected Behavior

When importing existing resource of type `azurerm_monitor_metric_alert`, all fields including `name`, `criteria` and `action` should be imported.

### Actual Behavior

Only `id`, `resource_group_name` and `tags` are imported. Below is output of `terraform state show ...`:

```
resource "azurerm_monitor_metric_alert" "example" {
    id                  = "/subscriptions/xxx/resourceGroups/xxx/providers/Microsoft.Insights/metricalerts/example-alert"
    resource_group_name = "..."
    tags                = {}

    timeouts {}
}
```

### Steps to Reproduce

1. Create an Azure Monitor metric alert beforehand via Portal or Azure CLI.
2. Import the resource in terraform.
3. Check the imported state, many fields will be missing.

**Comment by wenesak** [comment]
same here, I was trying to import some existing "metric alerts" and got the same problem as xsreality.

**Comment by magodo** [comment]
This should be fixed by #7159.

## Clarification

As the description of the PR #7159 said:

> Previous code was using the `SingleResourceMultipleMetricCriteria`, while this is superseded by `MultipleResourceMultipleMetricCriteria`.

In your case the metric of the target resource `azurerm_traffic_manager_profile` might be a `MultipleResourceMultipleMetricCriteria` (you can tell via looking at the field `properties.criteria.odata.type` from the response body), which is omit in current [code](https://github.com/terraform-providers/terraform-provider-azurerm/blob/04b3334c4320a188fa3d56ed27ae1c7499b26b5d/azurerm/internal/services/monitor/monitor_metric_alert_resource.go#L412).

**Comment by magodo** [comment]
I'm gonna close this issue as #7159 has been merged now. If the issue still exists, please feel free to reopen this issue.
